idb/postgres/internal/encoding/encode.go

```diff
@@ -73,3 +98,125 @@ func init() {
 	jsonCodecHandle.Indent = 0
 	jsonCodecHandle.MapKeyAsString = true
 }
+
+// EscapeNulls converts null bytes, invalid utf8, and backslashes into backslash escape sequences \uXXXX
```

Comment:
Is there anything we can use in `%q`? It seems very similar, we would have to strip the wrapped quotes and add an extra layer of escapes.
```
func TestQuote(t *testing.T) {
	invalid := "\xc0"
	quoted := fmt.Sprintf("%q", invalid)
	require.Equal(t, `"\xc0"`, quoted)
	unquoted, e := strconv.Unquote(quoted)
	require.NoError(t, e)
	require.Equal(t, invalid, unquoted)
}
```